openipam/network/migrations/0007_auto_20170719_1150.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2017-07-19 17:50
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('network', '0006_auto_20170324_1644'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Building',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('building_number', models.CharField(max_length=255, unique=True)),
+                ('description', models.TextField()),
+                ('changed', models.DateTimeField(auto_now=True)),
+                ('changed_by', models.ForeignKey(db_column=b'changed_by', on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='BuildingToVlan',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('changed', models.DateTimeField(auto_now=True)),
+                ('building', models.ForeignKey(db_column=b'building', on_delete=django.db.models.deletion.CASCADE, to='network.Building')),
+                ('changed_by', models.ForeignKey(db_column=b'changed_by', on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
+                ('vlan', models.ForeignKey(db_column=b'vlan', on_delete=django.db.models.deletion.CASCADE, to='network.Vlan')),
+            ],
+        ),
+        migrations.AddField(
```

Comment:
Is this adding a field to the VLAN table, or is this doing django trickery to create a bridge table?